### PR TITLE
 [MIG] partner_vat_unique: Migration to 12.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

File: partner_vat_unique/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 Grant Thornton Spain - Ismael Calvo <ismael.calvo@es.gt.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner VAT Unique",
```
Comment (nikul-serpentcs) [inline partner_vat_unique/__manifest__.py:4]: Add  `summary`

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: partner_vat_unique/__manifest__.py (hunk 1 above)
Comment (rven) [reply partner_vat_unique/__manifest__.py:4]: Done!

File: partner_vat_unique/tests/test_vat_unique.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+from odoo.tests.common import SavepointCase
+from odoo.exceptions import ValidationError
+
+
+class TestVatUnique(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestVatUnique, cls).setUpClass()
+        cls.partner = cls.env['res.partner'].create({
```
Comment (nikul-serpentcs) [inline partner_vat_unique/tests/test_vat_unique.py]: @rven `cls.env['res.partner']` object Use multiple time, Use as a Global Variable

**Review by nikul-serpentcs — APPROVED** [review]
Minor Change, Otherwise LGTM :+1:

File: partner_vat_unique/tests/test_vat_unique.py (hunk 2 above)
Comment (koenloodts) [reply partner_vat_unique/tests/test_vat_unique.py]: Done

**Review by dynapps — APPROVED** [review]
Tested and works fine!

**Comment by rven** [comment]
@pedrobaeza could this be merged to 12.0 branch?

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM. Ideas for further improvement: put the name of the other partner in the message on another partner already having the same vat. Precise feedback is very important.

One minor point: license and probably other url's could be https.

**Comment by pedrobaeza** [comment]
@rven are you going to implement the enhancements proposed by @NL66278 or should we continue as is?

**Comment by pedrobaeza** [comment]
@NL66278 you didn't give the opportunity to @rven to answer, but I suppose it can be added as a new PR if wanted.

**Comment by NL66278** [comment]
That was my idea as well.

**Pull request merged**